src/tools/interop/idt/scripts/activate.sh

```diff
@@ -0,0 +1,25 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+sudo docker run \
```

Comment:
Does this need to run as `sudo`? Is it for access to the hosts usb and dbus?

Reply:
Well with the instructions as they're written today, just connecting to daemon would require root... 

Might make sense to collapse these conversations to single thread here: https://github.com/project-chip/connectedhomeip/pull/29174#discussion_r1330805987

Reply:
We'll revisit this!